config/application.rb

```diff
@@ -56,5 +56,8 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
+
+    # Allow lvh.me for subdomain garbage
```

Comment:
The tutorial recommended using `lvh.me` (for ex, `dcaf.lvh.me`) to try out subdomain stuff in practice. Not sure if I like it as an approach or not, and esp not in `application.rb`, but can figure it out down the line.